scripts/bundles/compiler.ts

```diff
@@ -8,18 +8,18 @@ import type { OutputChunk, RollupOptions, RollupWarning, TransformResult } from
 import sourcemaps from 'rollup-plugin-sourcemaps';
 import { minify, MinifyOptions } from 'terser';
 
-import { getBanner } from '../utils/banner';
-import { getTypeScriptDefaultLibNames } from '../utils/dependencies-json';
-import type { BuildOptions } from '../utils/options';
-import { writePkgJson } from '../utils/write-pkg-json';
-import { aliasPlugin } from './plugins/alias-plugin';
-import { inlinedCompilerDepsPlugin } from './plugins/inlined-compiler-deps-plugin';
-import { parse5Plugin } from './plugins/parse5-plugin';
-import { replacePlugin } from './plugins/replace-plugin';
-import { sizzlePlugin } from './plugins/sizzle-plugin';
-import { sysModulesPlugin } from './plugins/sys-modules-plugin';
-import { terserPlugin } from './plugins/terser-plugin';
-import { typescriptSourcePlugin } from './plugins/typescript-source-plugin';
+import { getBanner } from '../utils/banner.js';
```

Comment:
Ok so this `.js` extension here is supported if we want this to 'just work'. Alternatively, we could leave this imports as they are and instead add the `--experimental-specifier-resolution=node` CLI flag to `node` whenever we call a script in this directory. Personally I don't like putting the file extension but realistically it's not _that_ big of a deal, and if we do so then the compiled files 'just work' in node without anything weird.